pom.xml

```diff
@@ -10,7 +10,7 @@
 	</parent>
 	<groupId>ca.on.oicr.icgc.argo</groupId>
 	<artifactId>program-service</artifactId>
-	<version>1.2.0</version>
+	<version>1.2.1</version>
```

Comment:
is this a hotfix ? I think it should be 1.3.0

Reply:
fair enough wil change